app.ts

```diff
@@ -11,12 +10,17 @@ const app = express();
 let CLIENT_DIR = 'client/build';
 
 if (path.basename(__dirname) === 'build') {
-  CLIENT_DIR = '../' + CLIENT_DIR;
+  CLIENT_DIR = path.join('..', CLIENT_DIR);
 }
 
 // Serve static files from the React app
 app.use(express.static(path.join(__dirname, CLIENT_DIR)));
-app.use(cors());
+
+if(process.env.NODE_ENV !== 'production') {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  app.use(require('cors'));
```

Comment:
I think conditional imports is one of the few cases where we can disable this `eslint` rule

Reply:
Fine by me 👍